app/models/deferred/visitor.rb

```diff
@@ -0,0 +1,4 @@
+module Deferred
+  class Visitor < ::Visitor
+  end
```

Comment:
It appeals to my literary side.  However, I think it might require too much interpretation on the part of the average developer...

My suggestion would be to give it an initialiser that raises an exception if anyone tries to define instance methods at this level, but otherwise passes them down (up?) the chain.  That might be overcomplicating it, though.